packages/instant-apps-components/src/utils/locale.ts

```diff
@@ -2,10 +2,12 @@
 import { loadModules } from '../utils/loadModules';
 import { languageMap } from './languageUtil';
 
-export function getComponentClosestLanguage(element: HTMLElement): string | undefined {
-  const closestElement = (element.closest('[lang]') as HTMLElement) ?? element.shadowRoot?.ownerDocument?.documentElement;
+const TEST_ENV_ORIGIN = 'localhost:5173';
```

Comment:
Is this only for testing language switcher or will it affect other apps?

Reply:
this is just to properly handle language assets in a test environment.

Reply:
but yes, the only component that this affects is language translator. this just grabs the value from the lang attribute on the html tag rather than scanning for the closest element

Reply:
I meant the `TEST_ENV_ORIGIN` the actual test env could be something like localhost:3000, localhost:3001, localhost:5173, localhost:5174 etc but here it's hardcoded to localhost:5173

Reply:
The `vitest` browser test environment is an isolated test environment that is served on `localhost:5173` by default. When running `vitest watch`

Reply:
These tests won't be run within the context of a web app

Reply:
When running vitest, this is the environment that the tests run in by default: 
<img width="1728" alt="image" src="https://github.com/user-attachments/assets/5602f817-b0b1-4ed4-ab36-161b38b3c366">